images/router/haproxy/reload-haproxy

```diff
@@ -69,6 +69,16 @@ function haproxyHealthCheck() {
 }
 
 
+function adjustOomScores() {
+    for pid; do
+	old_score=$(< /proc/$pid/oom_score_adj)
+	new_score=$((old_score + 10))
```

Comment:
Any concern that this (hypothetically) wraps which (assuming it goes negative) would defeat the intent here?
I didn't look at the underlying types - just a throwaway observation.

Reply:
The max value is 1000, so I'll add a check to avoid going above that.